mis_builder/models/mis_report_instance.py

```diff
@@ -407,7 +407,12 @@ def _get_additional_query_filter(self, query):
         Returns an Odoo domain expression (a python list)
         compatible with the model of the query."""
         self.ensure_one()
-        return []
+        domain = []
+        if company_field := query.sudo().company_field_id:
+            instance_companies = self.report_instance_id.query_company_ids
+            assert instance_companies
+            domain = [(company_field.name, "in", instance_companies.ids)]
```

Comment:
```suggestion
            query_company_ids = self.report_instance_id.query_company_ids.ids
            assert query_company_ids
            domain = [(company_field.name, "in", query_company_ids)]
```

Just a nitpick, but in general I prefer to keep variable names so we have a uniform vocabulary.

Reply:
Done :+1: